$NetBSD: patch-aa,v 1.5 2003/03/09 22:00:55 dmcmahill Exp $

first patch is for guile-1.6.3.
second is because (a) gnu m4 is needed for 1.6 and earlier NetBSD and probably
all other OS's. and (b) we want to support site, user, and project config files
the same way as the cad/pcb package.

--- scheme/gnet-PCBboard.scm.orig	Sun Jan 27 14:43:54 2002
+++ scheme/gnet-PCBboard.scm	Sun Mar  9 15:51:29 2003
@@ -139,5 +139,5 @@
 ;; To emulate popen. Guileish again.
 ; Needed after guile ver. 1.3.2. To save 1.3a users, wrap it in.
-(false-if-exception (use-modules (ice-9 popen)))
+(use-modules (ice-9 popen))
 
 (define (PCBboard output-filename)
@@ -152,5 +152,5 @@
 ;;  (let ((pipe (open-output-pipe (string-append "m4 " *m4-pcbdir* "/common.m4 - >> " output-filename))))
 ;; Fixed pipe command (AVH 1/27/02)
-   (let ((pipe (open-output-pipe (string-append "m4 -d -I" *m4-pcbdir* " " *m4-pcbdir* "/common.m4 - >> " output-filename))))
+   (let ((pipe (open-output-pipe (string-append "gm4 -d -I" *m4-pcbdir* " -I/usr/X11R6/etc/pcb -I$HOME/.pcb -I. " *m4-pcbdir* "/common.m4 - >> " output-filename))))
 
 
